action.yml

```diff
@@ -43,14 +43,13 @@ branding:
 runs:
   using: 'composite'
   steps:
+    - uses: actions/setup-python@v5.1.1
+      with:
+        python-version: '3.11'
+
     - name: Install Python dependencies
       run: |
-        python3 -m pip install --upgrade pip
-        python3 -m pip install -r ${{ github.action_path }}/requirements.txt
-      shell: bash
-
-    - name: Call command python3
-      run: python3 --version
+        python -m pip install -r ${{ github.action_path }}/requirements.txt
```

Comment:
`- run: pip install -r requirements.txt` should be enough.

Reply:
Addressed in commit 58756d11c9b8d7e6668d587599c9e41621e4b3a7/

Reply:
I did several experiments with that.
- When the setup-python cache option is used, it uses the host root path and even looks for the full path inside this root dir.
- I have returned to the original independent solution but have not started with the 'python' command.